Description: Fix test (Ruby2.2-specific syntax?)
Author: Lucas Nussbaum <lucas@debian.org>
Origin: vendor
Bug: https://github.com/vmg/redcarpet/issues/485
Forwarded: https://github.com/vmg/redcarpet/issues/485
Last-Update: 2015-05-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/markdown_test.rb
+++ b/test/markdown_test.rb
@@ -359,12 +359,12 @@
 
   def test_no_rewind_into_previous_inline
     result = "<p><em>!dl</em><a href=\"mailto:1@danlec.com\">1@danlec.com</a></p>\n"
-    output = render("_!dl_1@danlec.com", with: [:autolink])
+    output = render_with({:autolink => true}, "_!dl_1@danlec.com")
 
     assert_equal result, output
 
     result = "<p>abc123<em><a href=\"http://www.foo.com\">www.foo.com</a></em>@foo.com</p>\n"
-    output = render("abc123_www.foo.com_@foo.com", with: [:autolink])
+    output = render_with({:autolink => true}, "abc123_www.foo.com_@foo.com")
 
     assert_equal result, output
   end
